samples/Localization.CustomResourceManager/Startup.cs

```diff
@@ -55,7 +55,12 @@ public void ConfigureServices(IServiceCollection services)
                 new CultureInfo("zh"),
                 new CultureInfo("zh-CN")
             };
-
+#if NET461
+            supportedCultures.Add(new CultureInfo("zh-CHT"));
```

Comment:
I don't know.  The test was like this before the `net46` target was removed.  This change is just restoring the removed code, and changing the condition from `NET46` to `NET461`.